docker-compose.yml

```diff
@@ -110,6 +113,8 @@ services:
       REDIS_URL: # redis://cache:6379 - not required for single machine deployment
       GEOIPSERVER_URL: http://geoip:8080/json
       GOOGLE_CLIENT_ID: ${GOOGLE_CLIENT_ID}
+      MAXMIND_LICENSE_KEY: ${MAXMIND_LICENSE_KEY}  
```

Comment:
MAXMIND_LICENSEKEY: ${MAXMIND_LICENSE_KEY}  
MAXMIND_ACCOUNTID: ${MAXMIND_ACCOUNT_ID} 

@saalim-mushtaq update here web api service has slightly different name for this env